Templates/HttpPUT(CRUD)-CSharp/sample.dat

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Azure",
```

Comment:
Normalize the property casing. Since this is JSON, probably should all be camelCase.